Automattic/wp-calypso
# [WIP] Gutenberg: Fix Post Visibility styles

#### Changes proposed in this Pull Request

Fixes #27697.

The Post Visibility dialog in the Calypso Gutenberg editor has some styling issues caused by:
* Lack of the WP.org styles in the Gutenberg packages.
* Calypso is defining default styles that overrides the styles provided by the Gutenberg packages.

This PR adds the missing CSS rules for the `input[type="radio"]` and `label` elements, so the dialog looks like when it is rendered in the WP.org Gutenberg editor.

Calypso overrides are solved by using the styles from the `@wordpress/edit-post` package which apparently are re-declaring most of the Calypso default styles. These styles were previously imported from a local copy of the files in Calypso itself, but this copy has not been updated in a while. However, the copied files were modified in order to fix some of the styling issues detected in the past, so all this modifications are now moved to the `gutenberg/editor/style.scss` file.

#### Testing instructions

* Create a new post in Gutenberg
* Check on the sidebar that the Post Visibility dialog looks like the After screenshot below.
* Make sure that no new styling issues are present

**Before**
<img width="368" alt="46697681-67369080-cbd2-11e8-8923-252303a2e067" src="https://user-images.githubusercontent.com/1233880/47463478-a2e27480-d7e6-11e8-9cec-1047098b75a9.png">

**After**
<img width="294" alt="screen shot 2018-10-24 at 23 27 08" src="https://user-images.githubusercontent.com/1233880/47463453-93632b80-d7e6-11e8-8c06-cb00f7b3f213.png">
